lib/active_storage_validations/dimension_validator.rb

```diff
@@ -119,10 +119,10 @@ def is_valid?(record, attribute, file_metadata)
       true # valid file
     end
 
-    def add_error(record, attribute, type, *attrs)
+    def add_error(record, attribute, type, **attrs)
       key = options[:message].presence || type
       return if record.errors.added?(attribute, key)
-      record.errors.add(attribute, key, *attrs)
+      record.errors.add(attribute, key, **attrs)
```

Comment:
@igorkasyanchuk Could you maybe throw out a new release? Since this change is required to run with ruby 3.

Reply:
I'll release it today

Reply:
Successfully registered gem: active_storage_validations (0.9.2)

thanks again!